src/main/java/Main/Grepper.java

```diff
@@ -0,0 +1,38 @@
+package Main;
+
+import Arguments.GrepperArguments;
+import Arguments.PrintArguments;
+import Arguments.SearchArguments;
+import Arguments.TranscriptArguments;
+import Print.PrintAdapter;
+import Print.Printable;
+import Search.SearchAdapter;
+import Search.Searchable;
+import Transcription.TranscriptionAdapter;
+
+import java.util.List;
+
+public class Grepper {
+
+    GrepperArguments arguments;
```

Comment:
Don't need to store these. Have the adapters as instance variables instead!

Reply:
Done with lombok.